packages/form-builder/src/components/FormFieldList/FormFieldList.vue

```diff
@@ -10,9 +10,14 @@
 			{{ listTitle }}
 		</h3>
 
-		<p v-if="description">
-			{{ description }}
-		</p>
+		<slot
+			name="description"
+			v-bind="description"
```

Comment:
```suggestion
			v-bind="{ description }"
```
Sinon il y a une erreur `slot v-bind without argument expects an Object`